packages/botonic-nlp/tests/utils/model/model-config.json

```diff
@@ -0,0 +1 @@
+{"locale":"en","maxLength":12,"vocabulary":["<PAD>","<UNK>","i","looking","a","size","xxs","wool","belts","xs","hate","gray","linen","hat","hoodie","m","?","jacket","s","blue","leather","pink","t-shirt","fur","cotton","xl","shirt","l","love",".","xxl","red","can","someone","tell","where","buy","want","return","not","white","orange","sale","brown","allergic","black","material","jeans","understand","people","coats","clothes","new","trousers","who","wears","coat"],"entities":["O","product","color","material","size"]}
```

Comment:
in other projects we call "helpers" instead of "utils" the stuff useful for tests